guides/doc-Provisioning_Guide/topics/Infoblox_Integration.adoc

```diff
@@ -15,6 +15,8 @@ All DHCP and DNS records can be managed only in a single Network or DNS view, Af
 
 Hosting PXE-related files using Infoblox's TFTP functionality is not supported. You must use {SmartProxy} as a TFTP server for PXE provisioning. For more information, see xref:Configuring_Networking[].
 
+{Project} does not support using Infoblox IPAM.
```

Comment:
{Project} does not support using Infoblox as {Project} IPAM provider.

Reply:
I have updated it but will leave it as is for now downstream.